vendor/vendor.json

```diff
@@ -6,6 +6,46 @@
 			"path": "context",
 			"revision": ""
 		},
+		{
```

Comment:
mmm FYI we are very close to switching over to go modules, so this may become obsolete by the time it comes to actually merge this however managing dependencies with go modules is much easier and "the right thing" should be done automatically if you turn on go modules.